projects/media-viewer/src/lib/viewers/pdf-viewer/outline-view/outline-view.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'mv-outline-view',
+  templateUrl: './outline-view.component.html',
+  styleUrls: ['../../../styles/main.scss'],
```

Comment:
this needs to go... it can't live any longer with the component... check my css change branch

Reply:
will do after release

Reply:
done